articles/130_ros_time.md

```diff
@@ -140,6 +144,15 @@ The underlying datatypes will also provide ways to register notifications, howev
   <b>TODO:</b> Enumerate the <code>rcl</code> datastructures and methods here.
 </div>
 
+## Work Needing To Be Done (Apr 2017)
+
+- Decide where and when to autosubscribe to the `/clock` message. Built in to Node? A Node addon or service? Part of the first call to `rcl_node_init`?
+- Decide how to deal with multiple nodes in a single process that all update the one static time (or decide to ditch the static time methods).
+- Decide where to put the out-of-the-box clock publishers.
+- Ensure that there is no reason we can't enable `ROSTime` (with fallbacks) by default.
+- Change the RCL time implementation to call a method on the time source to return the current time (rather than using a field). This is necessary to compute the current time from the realtime factor. Fix the `rcl_get_ros_time` method to use the newly-added current time method.
+- Add an `rcl_time_source_t` (to the RCL project) for `ROSTime`.
```

Comment:
I attempted to clarify the API change.

The real time factor has to be included in the clock message. I've tried to clarify that in the document as well.